client/state/post-formats/reducer.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import { isValidStateWithSchema } from 'state/utils';
+import * as schema from './schema';
```

Comment:
Makes sense. It's a good time to rename the exported schema object to something more unique anyway.